### DISCUSSION TITLE
This Deployment has been disabled.

### DISCUSSION THREAD

**Discussion by sumedh1996** [vercel/community#2035] [Help]
### Summary

I am getting the following error on my production environment

This Deployment has been disabled.

Your connection is working correctly.

Vercel is working correctly.

451: DEPLOYMENT_DISABLED
Code: DEPLOYMENT_DISABLED
ID: bom1::j8s2t-1679632924722-6f75686ed354

If you are a visitor, contact the website owner or try again later.
If you are the owner, please contact support.
![screenshot_2023-03-29_at_12 08 23_am](https://user-images.githubusercontent.com/46312679/231070304-88f35750-6dc0-4447-939f-3d3fc15de757.png)

**Comment by amyegan** [comment]
Hi @sumedh1996! Please let me know the URL or ID of the deployment that is throwing this error so I can look into the cause 🙏

**Comment by sumedh1996** [comment]
@amyegan 
This was the deployment:
https://vercel.com/tealfeed/frontend-tealfeed/CmEPbGCnfiVUVMF1Tmho6yB5Y9y4

**Reply by amyegan** [reply]
Thanks for sharing the deployment link @sumedh1996. I don't see any block or other problem with that deployment. Are you still getting the error?

**Comment by sumedh1996** [comment]
@amyegan  This is a frequent issue for our production environment. Over the past 1 month, it happened 4-5 times and once I redeploy the master things go back to normal.  Currently, I am unable to find any sort of fix for the above issue.
